src/base/AnyMap.cpp

```diff
@@ -5,6 +5,10 @@
 
 #include "cantera/base/AnyMap.h"
 
+#ifdef CANTERA_API_NO_BOOST
+#include "cantera/base/AnyMap.inl.h"
+#endif
```

Comment:
This should be unnecessary, right? `CANTERA_API_NO_BOOST` should never be defined when compiling Cantera, so the `.inl` file should have been included via `AnyMap.h`.

Reply:
Yep. This was an artifact from when I thought I need to do something at compile time. Removed!